### PR TITLE
[Docs] Add log format section

### PR DESCRIPTION
**Pull request by def-**
Adapted from https://github.com/google/glog/blob/d8cb47f77d1c31779f3ff890e1a5748483778d6a/src/glog/logging.h.in#L265-L292 which is the glog version (0.3.4) we use for logging in master & tserver.

### CODE REVIEW

File: docs/content/preview/troubleshoot/nodes/check-logs.md [hunk 1]
```diff
@@ -56,3 +56,22 @@ For YB-Master and YB-TServer, the log rotation size is controlled by the `--max_
 For YSQL, you also have the additional `postgres*log` files. These logs have daily and size-based log rotation, that is a new log file will be created each day or a log reaches 10 MB size.
 
 For available configuration flags, see [YB-Master logging flags](../../../reference/configuration/yb-master/#logging-flags) and [YB-TServer logging flags](../../../reference/configuration/yb-tserver/#logging-flags).
+
+## Log format
+
+YB-Master and YB-TServer log messages have this form:
+
+```
+Lmmdd hh:mm:ss.uuuuuu threadid file:line] msg
```
Comment (polarweasel) [inline docs/content/preview/troubleshoot/nodes/check-logs.md:65]: Missing a left square bracket here. Which parts are optional?

File: docs/content/preview/troubleshoot/nodes/check-logs.md [hunk 2]
```diff
@@ -56,3 +56,22 @@ For YB-Master and YB-TServer, the log rotation size is controlled by the `--max_
 For YSQL, you also have the additional `postgres*log` files. These logs have daily and size-based log rotation, that is a new log file will be created each day or a log reaches 10 MB size.
 
 For available configuration flags, see [YB-Master logging flags](../../../reference/configuration/yb-master/#logging-flags) and [YB-TServer logging flags](../../../reference/configuration/yb-tserver/#logging-flags).
+
+## Log format
+
+YB-Master and YB-TServer log messages have this form:
+
+```
```
Comment (polarweasel) [inline docs/content/preview/troubleshoot/nodes/check-logs.md:64]:
```suggestion
```output
```

File: docs/content/preview/troubleshoot/nodes/check-logs.md [hunk 3]
```diff
@@ -56,3 +56,22 @@ For YB-Master and YB-TServer, the log rotation size is controlled by the `--max_
 For YSQL, you also have the additional `postgres*log` files. These logs have daily and size-based log rotation, that is a new log file will be created each day or a log reaches 10 MB size.
 
 For available configuration flags, see [YB-Master logging flags](../../../reference/configuration/yb-master/#logging-flags) and [YB-TServer logging flags](../../../reference/configuration/yb-tserver/#logging-flags).
+
+## Log format
+
+YB-Master and YB-TServer log messages have this form:
```
Comment (polarweasel) [inline docs/content/preview/troubleshoot/nodes/check-logs.md:62]:
```suggestion
YB-Master and YB-TServer log messages follow this pattern:
```

File: docs/content/preview/troubleshoot/nodes/check-logs.md [hunk 4]
```diff
@@ -56,3 +56,22 @@ For YB-Master and YB-TServer, the log rotation size is controlled by the `--max_
...
+
+```
+Lmmdd hh:mm:ss.uuuuuu threadid file:line] msg
+```
+
+The fields are defined as follows:
```
Comment (polarweasel) [inline docs/content/preview/troubleshoot/nodes/check-logs.md:68]:
```suggestion
The fields are as follows:
```

File: docs/content/preview/troubleshoot/nodes/check-logs.md [hunk 5]
```diff
@@ -56,3 +56,22 @@ For YB-Master and YB-TServer, the log rotation size is controlled by the `--max_
...
+
+- `L`: A single character, representing the log level (`I` for INFO, `W` for WARNING, `E` for ERROR, `F` for FATAL)
+- `mm`: Month (zero padded; i.e. May is `05`)
+- `dd`: Day (zero padded)
+- `hh:mm:ss.uuuuuu`: Time in hours, minutes and fractional seconds
+- `threadid`: Thread ID
+- `file`: File name
+- `line`: Line number
+- `msg`: The logged message
```
Comment (polarweasel) [inline docs/content/preview/troubleshoot/nodes/check-logs.md:77]:
```suggestion
- `L`: A single character, representing the log level (`I` for INFO, `W` for WARNING, `E` for ERROR, and `F` for FATAL)
- `mm`: Month (zero-padded; for example, May is `05`)
- `dd`: Day (zero-padded)
- `hh:mm:ss.uuuuuu`: Time in hours, minutes, and fractional seconds
- `threadid`: Thread ID
- `file`: File name
- `line`: Line number
- `msg`: The logged message
```

File: docs/content/preview/troubleshoot/nodes/check-logs.md (hunk 1 above)
Comment (def-) [reply docs/content/preview/troubleshoot/nodes/check-logs.md:65]: There is no left square bracket, that's the actual log format. There are no optional parts.

**Comment by def-** [comment]
@polarweasel Done.

**Pull request merged**